boards/arm/pan1770_evb/pan1770_evb.dts

```diff
@@ -0,0 +1,257 @@
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "pan1770_evb-pinctrl.dtsi"
+
+/ {
+	model = "pan1770-evb";
+	compatible = "panasonic-industrial-devices-europe-gmbh,pan1770-evb";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		evb_led1: evb_led_1 {
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+			label = "LED1 on EVB";
+		};
+		evb_led2: evb_led_2 {
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+			label = "LED2 on EVB";
+		};
+		evb_led3: evb_led_3 {
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+			label = "LED3 on EVB";
+		};
+		evb_led4: evb_led_4 {
+			gpios = <&gpio0 16 GPIO_ACTIVE_LOW>;
+			label = "LED4 on EVB";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_evb_led1: pwm_evb_led_1 {
+			pwms = <&pwm0 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		evb_sw1: evb_sw_1 {
+			gpios = <&gpio0 11 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW1 on EVB";
+		};
+		evb_sw2: evb_sw_2 {
+			gpios = <&gpio0 12 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW2 on EVB";
+		};
+		evb_sw3: evb_sw_3 {
+			gpios = <&gpio0 24 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW3 on EVB";
+		};
+		evb_sw4: evb_sw_4 {
+			gpios = <&gpio0 25 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW4 on EVB";
+		};
+	};
+
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio0 3 0>,   /* A0 */
+				   <1 0 &gpio0 4 0>,   /* A1 */
+				   <2 0 &gpio0 28 0>,  /* A2 */
+				   <3 0 &gpio0 29 0>,  /* A3 */
+				   <4 0 &gpio0 30 0>,  /* A4 */
+				   <5 0 &gpio0 31 0>,  /* A5 */
+				   <6 0 &gpio1 1 0>,   /* D0 */
+				   <7 0 &gpio1 2 0>,   /* D1 */
+				   <8 0 &gpio1 3 0>,   /* D2 */
+				   <9 0 &gpio1 4 0>,   /* D3 */
+				   <10 0 &gpio1 5 0>,  /* D4 */
+				   <11 0 &gpio1 6 0>,  /* D5 */
+				   <12 0 &gpio1 7 0>,  /* D6 */
+				   <13 0 &gpio1 8 0>,  /* D7 */
+				   <14 0 &gpio1 10 0>, /* D8 */
+				   <15 0 &gpio1 11 0>, /* D9 */
+				   <16 0 &gpio0 12 0>, /* D10 */
+				   <17 0 &gpio0 13 0>, /* D11 */
+				   <18 0 &gpio0 14 0>, /* D12 */
+				   <19 0 &gpio0 15 0>, /* D13 */
+				   <20 0 &gpio0 26 0>, /* D14 */
+				   <21 0 &gpio0 27 0>; /* D15 */
+	};
+
+	arduino_adc: analog-connector {
+		compatible = "arduino,uno-adc";
+		#io-channel-cells = <1>;
+		io-channel-map = <0 &adc 1>,	/* A0 = P0.3 = AIN1 */
+						 <1 &adc 2>,	/* A1 = P0.4 = AIN2 */
+						 <2 &adc 4>,	/* A2 = P0.28 = AIN4 */
+						 <3 &adc 5>,	/* A3 = P0.29 = AIN5 */
+						 <4 &adc 6>,	/* A4 = P0.30 = AIN6 */
+						 <5 &adc 7>;	/* A5 = P0.31 = AIN7 */
```

Comment:
nit: alignment is still wrong here, is your editor configured correctly? tab width should be 8

Reply:
Oh that's strange, i thought i got the correct tab size now and it looked good in my editor. I will check this again. Thanks!

Reply:
Tab width is now definitely set to 8. So it's looking good in all of my editors. And I took the clean nRF52840dk_nrf52840.dts from zephyr repo as an example, which using the same tabs and spaces and looks also good in editor.